org.eclipse.scout.rt.platform/src/main/java/org/eclipse/scout/rt/platform/nls/NlsUtility.java

```diff
@@ -68,32 +70,90 @@ public static String bindText(String text, String... messageArguments) {
 
   /**
    * Applies all {@link ITextPostProcessor text post processors} registered in the {@link IBeanManager} to the text
-   * given. It is assumed that the given text uses the {@link NlsLocale}.
+   * given. It is assumed that the given text uses the {@link NlsLocale}. No {@code textKey} is used.
    *
    * @param text
    *          The text to post-process. May be {@code null}.
    * @return The text with all post-processing applied.
-   * @see #postProcessText(Locale, String)
-   * @see #postProcessText(Locale, String, Collection)
+   * @see #postProcessText(Locale, String, String, Collection, String...)
    */
   public static String postProcessText(String text) {
-    return postProcessText(NlsLocale.get(), text);
+    return postProcessText(NlsLocale.get(), null, text);
```

Comment:
is this method currently in use? otherwise we could deprecate it and instead use the overload using text and key?